cli/src/test/java/de/jplag/cli/test/CliArg.java

```diff
@@ -0,0 +1,21 @@
+package de.jplag.cli.test;
+
+public record CliArg<T>(String name, boolean isPositional) {
+    public static CliArg<String[]> SUBMISSION_DIRECTORIES = new CliArg<>("", true);
```

Comment:
Do we want to have every CLI arg as a constant here? Another option I could imagine working is, generating them from the https://github.com/jplag/JPlag/blob/feature/cliTests/cli/src/main/java/de/jplag/cli/options/CliOptions.java Class and then referencing them by there names in the methods were you now give a CliArg Object. 
I dont really have a preference, just wanted to leave that as an option for discussion here